sdk/storage/storage-blob/src/AppendBlobURL.ts

```diff
@@ -1,10 +1,11 @@
 import { HttpRequestBody, TransferProgressEvent } from "@azure/ms-rest-js";
 
-import * as Models from "../src/generated/lib/models";
+import * as Models from "../src/generated/src/models";
```

Comment:
[NIT] This can simply be "./generated/src/models" ?
[Same in every file]

Reply:
Yes, it should be.